### PR TITLE
Revise `dd` command to clarify and avoid misinterpretation in bootable-usb.

### LINKED ISSUE #968: Documentation - Create a bootable USB drive
Unless I am mistaken of=/dev/sdx and not /dev/sdx1. The documentation is unclear. Moreover #lsblk -S is a better bet than just #lsblk  
[xxx@amphib Downloads]$ lsblk -S
NAME HCTL       TYPE VENDOR   MODEL             REV TRAN
sda  3:0:0:0    disk WD       My_Passport_0810 1049 usb
sdb  2:0:0:0    disk WD       My_Passport_25E2 4005 usb
sdc  4:0:0:0    disk PNY      USB_3.0_FD       PMAP usb

### PR DESCRIPTION
**Pull request by mvincerx**
- Closes #968

Signed-off-by: Michael Vincerra <michael.vincerra@intel.com>

### CODE REVIEW

File: source/get-started/bootable-usb.rst [hunk 1]
```diff
@@ -88,11 +92,11 @@ Burn the |CL| image onto a USB drive
 
       sudo umount /dev/sdd*
 
-#. Burn the image onto the USB drive. The example below burns an image onto `<your USB device>`:
+#. Burn the image onto the USB drive. This example burns an image onto `/dev/sdd`.  The parent drive number of the usb varies.
```
Comment (bktan8) [inline source/get-started/bootable-usb.rst]: I don't understand what "The parent drive number of the usb varies." is trying to say.

Comment (bktan8) [reply source/get-started/bootable-usb.rst]: @mvincerx - we should "spell out" the example USB device for the MacOS portion too.  

Current:
sudo dd if=./clear-[version number]-live-[desktop | server].iso of=**<your USB device>** bs=4m

Change:
sudo dd if=./clear-[version number]-live-[desktop | server].iso of=/dev/disk2 bs=4m

Comment (mvincerx) [reply source/get-started/bootable-usb.rst]: I will replace that sentence (The parent drive..) with `The device name will vary. `

Will follow through with changes to Mac section. Good call.

Comment (mvincerx) [reply source/get-started/bootable-usb.rst]: @bktan8 See latest commit.

File: source/get-started/bootable-usb.rst [hunk 2]
```diff
@@ -88,11 +92,11 @@ Burn the |CL| image onto a USB drive
 
       sudo umount /dev/sdd*
 
-#. Burn the image onto the USB drive. The example below burns an image onto `<your USB device>`:
+#. Burn the image onto the USB drive. This example burns an image onto `/dev/sdd`.  The device name of the USB may vary.
```
Comment (bktan8) [inline source/get-started/bootable-usb.rst]: @mvincerx - would be better to say "Your USB device name may vary."?  Simply saying "The device name..." is too vague IMO.

Comment (mvincerx) [reply source/get-started/bootable-usb.rst]: Sure. NP.

Comment (bktan8) [reply source/get-started/bootable-usb.rst]: @mvincerx - keep as is and ship it.  :)

**Comment by mvincerx** [comment]
LGTM.

**Pull request merged**